domains/pf1x3r.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Milo40",
+           "email": "33132706+Milo40@users.noreply.github.com",
+           "discord": "411986766166491136"
+        },
+    
+        "record": {
+            "CNAME": "pf1.dev"
```

Comment:
Seems to just be a starting next.js template, please add more stuff to your website